### PR TITLE
Update adaptive tools

### PR DESCRIPTION
**Pull request by Danieladu**
1. Update LG/LU SDK version
2. Refine README
3. Embedded launch.json
4. Add structure code suggestion
5. Add LU code suggestion
6. Other small fixes
7. Update version to 0.0.3

### CODE REVIEW

File: experimental/adaptive-tool/README.md [hunk 1]
```diff
@@ -12,32 +12,78 @@
...
+- support LG template breaking points
+- support Expression breaking points
 
 ### Syntax highlighting, auto-suggest, auto complete functionality
 #### .lu documents
-- Syntax highlighting
+- syntax highlighting
+![lu_syntax_highlighting](./resources/images/lu_syntax_highlighting.png)
+- diagnostic check
+![lu_diagnostic](./resources/images/lu_diagnostic.png)
```
Comment (feich-ms) [inline experimental/adaptive-tool/README.md:26]: add lu auto completion?

File: experimental/adaptive-tool/README.md [hunk 2]
```diff
@@ -12,32 +12,79 @@
...
 ## Features
-### Debugging adaptive dialogs
-- Supporting adaptive dialog breaking point
-
-### Debugging language generation
-- Support LG template breaking points
-- Support Expression breaking points
+### Debugging
+[Setting up](#set_up)
+- supporting adaptive dialog breaking point
```
Comment (vishwacsena) [inline experimental/adaptive-tool/README.md:17]: Re-word to 

- Supports break points in .dialog files (adaptive dialog)
- Supports break points in .lg files
- Supports break points in adaptive expressions

File: experimental/adaptive-tool/README.md [hunk 3]
```diff
@@ -12,32 +12,79 @@
...
+### setting up
+To configure Visual Studio Code you need to add a target in your launch.settings file.
+
+You can do that by the **add a configuration** in the debug panel.
+
+There should be 2 configuration templates available:
+
+* **Bot: Launch .NET Core Configuration** - Configuration for building and launching your bot via **dotnet run** and connecting to it
+* **Bot: Attach Configuration** - Configuration for attaching to the debug port of an already running bot (such as IIS express)
+
```
Comment (vishwacsena) [inline experimental/adaptive-tool/README.md:69]: Please add a sample of what the completed configuration should look like.

File: experimental/adaptive-tool/README.md [hunk 4]
```diff
@@ -12,32 +12,79 @@
...
+### Using
+
+* Open any source file (*.dialog, *.lg) and set breakpoints.
+* Hit F5 to start debugger.
+
+As you interact with the bot your breakpoint should hit and you should be able to inspect memory, call context, etc.
```
Comment (vishwacsena) [inline experimental/adaptive-tool/README.md:75]: Add a screen shot showing/ highlighting where to inspect memory, call context etc.

**Pull request merged**